src/System.Private.ServiceModel/tests/Scenarios/Extensibility/WebSockets/WebSocketTests.cs

```diff
@@ -234,4 +234,230 @@ public static void WebSocketHttpsDuplexBinaryStreamed()
             clientReceiver.Dispose();
         }
     }
+
+    [Fact]
+    [OuterLoop]
+    [ActiveIssue(470)]
+    public static void WebSocketHttpsDuplexTextStreamed()
+    {
+        TextMessageEncodingBindingElement textMessageEncodingBindingElement = null;
+        HttpsTransportBindingElement httpsTransportBindingElement = null;
+        CustomBinding binding = null;
+        ClientReceiver clientReceiver = null;
+        InstanceContext context = null;
+        DuplexChannelFactory<IWSDuplexService> channelFactory = null;
+        IWSDuplexService client = null;
+        FlowControlledStream uploadStream = null;
+
+        try
+        {
+            // *** SETUP *** \\
+            textMessageEncodingBindingElement = new TextMessageEncodingBindingElement();
+            httpsTransportBindingElement = new HttpsTransportBindingElement();
```

Comment:
Nit: not required for this PR, but I would like to start setting a max timeout on all our scenario tests.  In Linux testing where these fail (primarily WebSockets), the default timeout is tragically long.  Ideally we can use the test helpers MaxTestTimeout (sp?) so we can configure it during debugging.